Sources/PushNotifications.swift

```diff
@@ -145,8 +148,10 @@ import Foundation
 
         let persistenceService: InterestPersistable = PersistenceService(service: UserDefaults(suiteName: "PushNotifications")!)
 
+        let interestPersisted = persistenceService.persist(interest: interest)
```

Comment:
This variable name makes it look like persisting was successful. When actually it's more about `interestsChanged`...

Reply:
Would the variable name `isInterestPersisted` be better?

Reply:
No, because that's not what that boolean is.

It's true if they are different. If they are the same, they are persisted.